release_tester/selenium_ui_test/pages/views_page.py

```diff
@@ -294,17 +272,22 @@ def create_improved_views(self, view_name, types):
         sorted_value_sitem = self.locator_finder_by_xpath(sorted_value)
         sorted_value_sitem.click()
         time.sleep(2)
+        self.wait_for_ajax()
 
-        # print(f'Select stored field for {view_name} \n')
-        # # stored_field = "//div[contains(@id,'s2id_field')]"
-        # stored_field = "/html/body/div[1]/div/div[2]/div[2]/div/div[2]/div/table/tbody/tr/th/table/tbody/tr/td[1]/div"
-        #
-        # stored_field_sitem = self.locator_finder_by_xpath(stored_field)
-        # stored_field_sitem.click()
-        # stored_field_sitem.clear()
-        # stored_field_sitem.send_keys('attr')
-        # stored_field_sitem.send_keys(Keys.ENTER)
-        # time.sleep(2)
+        if self.current_package_version() >= semver.VersionInfo.parse("3.9.0"):
+            print('stored value has been skipped.\n')
+        else:
+            print(f'Select stored field for {view_name} \n')
+            # stored_field = "//div[contains(@id,'s2id_field')]"
+            stored_field = "/html/body/div[1]/div/div[2]/div[2]/div/div[2]/div/table/tbody/tr/th/table/tbody/tr/td[1]/div"
```

Comment:
there is no other way than this lengthy xpath?
It would as well be prefered, if you find a surrounding element that is better identifieable, and go from there up/down the xpath fewer steps.